scripts/validation/validate-no-duplicate-dirs.py

```
@@ -0,0 +1,229 @@
+        ".pytest_cache",
+        "archived",
+        "archive",
+        "node_modules",
+        ".venv",
+        "venv",
+        ".mypy_cache",
+        ".ruff_cache",
+        "dist",
+        "build",
+        "*.egg-info",
+    }
```

Ruff (0.14.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)